docs/hub/spaces-sdks.md

```diff
@@ -10,21 +10,21 @@ Gradio and Streamlit make it easy to quickly build apps using Python, and we rec
 
 ## Gradio
```

Comment:
Maybe we should then split this page into three, `spaces-sdks`, `spaces-gradio` and `spaces-streamlit, with `sdks` working as an index one

Reply:
Sounds good to me!

Reply:
Nice! I think if we split this into three pages we can have some more content on each page. For example, I really like how the [Tailwind docs](https://tailwindcss.com/docs/installation/framework-guides) has a bunch of framework-specific guides. I think we can do something similar with Gradio, Streamlit, and static HTML and do a super simple guide to setting up an app on the Hub.

Reply:
I added a Hot Dog Classifier to both the Gradio and Streamlit pages!